controllers/v1/solutions.js

```diff
@@ -1033,25 +1033,25 @@ module.exports = class Solutions extends Abstract {
    * @returns {JSON}
    */
 
-  async read(req) {
```

Comment:
@ankitpws this api won't be used anywhere after 5.2 right? confirmed that

Reply:
yes @VISHNUDAS-tunerlabs